Documentation Jorge Valdebenito summer 2022

JorgeValde opened the pull request.

JorgeValde committed:
Update Python Script

I update the documentation with the kfunctions python scripts and took notes on the changes in some files.

JorgeValde committed:
Update on the Python script chain

Update on the python chain

JorgeValde committed:
Create Documentation of JV for Summer 2022

This is a summary of the work during the summer of 2022

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,194 @@
+Most of the work was made in Python using the Spyder app. 
+
+The topics I worked are the following:
+1)	Creating a working environment in Python 
```
Your numbering is hard to follow because you use "1), 2),..." for the highest-level breakdown of tasks, but then you re-use it within "1)". You can just change to something different, like Roman numerals, a), b),... or whatever else looks good to you,

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,194 @@
+Third, we are creating the clusters by running simulations. 
+Fourth, we are calculating the agglomerations effects. 
+
+Next, the python scripts should be run in the following order:
```
This "Next" is confusing with the First, Second, etc. list above it. It would be clearer to replace it with something like, "In order to carry out the above four tasks,"

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,194 @@
+
+"G:...\ramosRivera\T-Burk\PngData\ScanData\Page_.jpg" (from Pdf2Jpg.py) --> OCR.py --> "G:...\PngData\OCR_Output_1998\OCR_Results.txt"
+
+pngwork.py imports several other datasets, I am only documenting the ones that are used in the program. 
```
Can you be clearer here? used in *what* program? Is this to say that pngwork.py imports datasets that it doesn't itself use?

kbuzard reviewed (changes requested):
Here are a few of my comments...please integrate with the two bigger picture issues we talked about today.

JorgeValde committed:
more detailed description 

I fixed the comments you mentioned in the last meeting. 
I still have to write instructions on how to construct a general environment for every user.

JorgeValde committed:
Update the order of the scripts

I updated this file with what we had before and following the comments from our last meeting

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,204 @@
+"G:...\T-Burk\PngData\cattell-all.dta"       --> pngwork.py --> 
+"G:...\T-Burk\PngData\Cattell_corr_list.dta" --> pngwork.py --> "G:...\T-Burk\PngData\cattLabs97.csv"
+
+I do not know where the dataset "Cattell_corr_list.dta" is created. 
```
@JorgeValde @Kirs10-Riley has this in her notes--essentially, it was emailed to me by Ivan Png. But please get the more detailed description from Kirsten

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,204 @@
+"G:...\T-Burk\PngData\cattLabs97.csv"       --> state_code_rep.py --> 
+"G:...\T-Burk\PngData\cattell_1997_raw.dta" --> state_code_rep.py --> "G:...\T-Burk\PngData\corr_cattLabs97.csv"
+
+I do not know where the dataset "cattell_1997_raw.dta" is created.
```
@Kirs10-Riley Do you know anything about this dataset that could be added here?

kbuzard commented on heterogeneity/Documentation_JV_Summer2022:
```diff
@@ -0,0 +1,204 @@
+"G:...\T-Burk\PngData\field.dta"           --> field_org.py -->
+"G:...\T-Burk\PngData\field-master.dta"    --> field_org.py --> "G:...\T-Burk\\LabData\\cal_lab_fields\\" + field"
+
+I do not know where the dataset "field.dta" is created.
+I do not know where the dataset "field-master.dta" is created.
```
@JorgeValde Again, I think that @Kirs10-Riley has documented these files.

kbuzard commented on heterogeneity/Python_Order:
```diff
@@ -1,103 +1,60 @@
 This file will document how to install and run the Python Scripts in the RamosRivera folder necessary for this project.
+This enviroment is only been created for the G drive in my personal computer. 
```
@JorgeValde This is confusing. What do you mean by personal computer? Isn't it true that if something is on the G:drive, everyone can access it? G: is the shared drive.

kbuzard reviewed (commented):
In response to your email: I got a notification of the pull request and merge. I've made a few comments here; I'm also going to look at what the files look like on my computer (instead of looking at the pull request, where it's a little hard to see the full layout of the files). I'll let you know if I have any further comments, but I don't have time to do this right now.
